packages/php-wasm/universal/src/lib/universal-php.ts

```diff
@@ -256,6 +256,15 @@ export interface IsomorphicLocalPHP extends RequestHandler {
 	 */
 	mv(oldPath: string, newPath: string): void;
 
+	/**
+	 * Copies a file or directory in the PHP filesystem to a
+	 * new location.
```

Comment:
Let's thoroughly explain what this function does in different situations, like when the target path exists doesn't exist.

Reply:
https://github.com/WordPress/wordpress-playground/blob/cc576fc972ceaabeba7660aaa39494c95aca2de8/packages/php-wasm/universal/src/lib/universal-php.ts#L259-L275

Reply:
Thank you! As I'm read it, I have additional questions:

> The target directory's parent must be a  valid location

A `valid location` means only that it must exist, right? If yes, let's say it directly.

>  If the target path is a file that already exists, it will be overwritten. 

Is that the case only when the source path is a file? Or also when it's a directory?

> If the target path is a directory that already exists, the file or directory will be copied into it.

So if the target path is a directory that does not exist yet, will it be created?

Reply:
I also wonder about the semantics here. Right now this function makes an implicit assumption that says:

> If the `newPath` is a directory that already exists, the developer intended to copy the `oldPath` inside that directory.

I don't think it always holds. Maybe the `newPath` shouldn't actually be there? I'd rather remove the implicit assumption and make the logic explicit: If `newPath` is a directory that already exists, that's an error, please remove it explicitly before the recursive copy.